src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/LocalAppContextSwitches.netfx.cs

```diff
@@ -7,38 +7,27 @@
 
 namespace Microsoft.Data.SqlClient
 {
-    internal static class LocalAppContextSwitches
+    internal static partial class LocalAppContextSwitches
     {
-        internal const string MakeReadAsyncBlockingString = @"Switch.Microsoft.Data.SqlClient.MakeReadAsyncBlocking";
-        private static int _makeReadAsyncBlocking;
-        public static bool MakeReadAsyncBlocking
-        {
-            [MethodImpl(MethodImplOptions.AggressiveInlining)]
-            get
-            {
-                return LocalAppContext.GetCachedSwitchValue(MakeReadAsyncBlockingString, ref _makeReadAsyncBlocking);
-            }
-        }
-
         internal const string UseMinimumLoginTimeoutString = @"Switch.Microsoft.Data.SqlClient.UseOneSecFloorInTimeoutCalculationDuringLogin";
-        private static int _useMinimumLoginTimeout;
+        private static bool _useMinimumLoginTimeout;
         public static bool UseMinimumLoginTimeout
         {
             [MethodImpl(MethodImplOptions.AggressiveInlining)]
             get
             {
-                return LocalAppContext.GetCachedSwitchValue(UseMinimumLoginTimeoutString, ref _useMinimumLoginTimeout);
+                return AppContext.TryGetSwitch(UseMinimumLoginTimeoutString, out _useMinimumLoginTimeout) ? _useMinimumLoginTimeout : false;
             }
         }
 
         internal const string DisableTNIRByDefaultString = @"Switch.Microsoft.Data.SqlClient.DisableTNIRByDefaultInConnectionString";
-        private static int _disableTNIRByDefault;
+        private static bool _disableTNIRByDefault;
         public static bool DisableTNIRByDefault
         {
             [MethodImpl(MethodImplOptions.AggressiveInlining)]
             get
             {
-                return LocalAppContext.GetCachedSwitchValue(DisableTNIRByDefaultString, ref _disableTNIRByDefault);
+                return AppContext.TryGetSwitch(DisableTNIRByDefaultString, out _disableTNIRByDefault) ? _disableTNIRByDefault : false;
```

Comment:
Can we change it to : return AppContext.TryGetSwitch(DisableTNIRByDefaultString, out _disableTNIRByDefault) && _disableTNIRByDefault;

Reply:
Same reply as above.